xtas/tasks/_alpino.py

```diff
@@ -113,7 +113,7 @@ def get_dep(line):
                 tokens=list(tokens.values()))
 
 
-def interpret_token(lemma, word, begin, _end, major_pos, _pos2, pos):
```

Comment:
Here we have major_pos, _pos2, pos, and major, which gets rather confusing. Maybe this is just Alpino terminology and we can't do much about it, but if we can, we should I think.